modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/fragment/renderer/SpaceSettingsComponentSectionFragmentRenderer.java

```diff
@@ -44,8 +51,30 @@ protected Map<String, Object> getProps(
 		FragmentRendererContext fragmentRendererContext,
 		HttpServletRequest httpServletRequest) {
 
+		ThemeDisplay themeDisplay =
+			(ThemeDisplay)httpServletRequest.getAttribute(
+				WebKeys.THEME_DISPLAY);
+
+		JSONArray companyAvailableLanguagesJSONArray =
```

Comment:
@julienmouraddotcom, most likely Brian will change to only jsonArray to simplify. I'm good with the changes, just leaving the comment